pkgs/test/default.nix

```diff
@@ -51,6 +51,23 @@ with pkgs;
 
   php = recurseIntoAttrs (callPackages ./php {});
 
+  pkg-configPackages =
+    let
+      # pkg-configPackages test needs a Nixpkgs with allowUnsupportedPlatform
+      # in order to filter out the unsupported packages without throwing any errors
+      # tryEval would be too fragile, masking different problems as if they're
+      # unsupported platform problems.
+      allPkgs = import ../top-level {
```

Comment:
```suggestion
      allPkgs = import pkgs.path {
```

Reply:
That risks copying a path to the store. `pkgs.path` is too complicated for me; most imports happen by path expression, which is known to work well.